camel-core/src/main/java/org/apache/camel/CamelContext.java

```diff
@@ -1206,6 +1206,14 @@
     DataFormat resolveDataFormat(String name);
 
     /**
+     * Resolve a data format given its name
```

Comment:
Does it resolve or create a new data format? Maybe try to specify what is the difference between the resolve vs create methods on CamelContext as it will confuse users.